copier.yml

```diff
@@ -27,62 +27,66 @@ odoo_version:
 
 org_slug:
   type: str
-  default: OCA
+  default: 360ERP
   help:
     What's the organization slug? If you are creating
     https://github.com/OCA/server-tools, then write "OCA" here.
+  when: false
 
 org_name:
   type: str
-  default: Odoo Community Association (OCA)
+  default: 360 ERP
   help:
     Tell me the Organization name. It's supposed to be human-readable. It will be used
-    in the author key of the __manifest__ files.
+    in the auethor key of the __manifest__ files.
+  when: false
 
 repo_slug:
   type: str
   help:
-    What's the repo slug? If you are creating https://github.com/OCA/server-tools, then
-    write "server-tools" here.
+    What's the repo slug? If you are creating https://github.com/360ERP/customer, then
+    write "customer" here.
 
 repo_name:
   type: str
+  default: "Customizations for {{ repo_slug }}"
   help:
-    Tell me the project name. It's supposed to be human-readable. So, server-tools
-    project could be named like "Tools for server environment(s)"
+    Tell me the project name. It's supposed to be human-readable. So, customer
+    project could be named like "Customizatons for Customer"
+  when: false
 
 repo_website:
   type: str
   default: "https://github.com/{{ org_slug }}/{{ repo_slug }}"
   help:
     Tell me the project website. It will be used in the website key of the __manifest__
     files.
+  when: false
 
 repo_description:
   type: str
+  default: "Customizations for {{ repo_slug }}"
   help: Please write a short description about what this repo is about.
+  when: false
 
 ci:
   type: str
-  default: GitHub
+  default: None
```

Comment:
Just for my information: why you change the default here?

Reply:
Oh, I would love to start running tests on 360ERP repos! But at this moment, the job would fail because it needs to be adapted to the 360ERP repo layout. Can we make this a follow-up ticket? Also, it will greatly increase the Github bill every month so maybe business should be involved?